server/app/models/ApplicantModel.java

```diff
@@ -38,6 +40,26 @@
 @Entity
 @Table(name = "applicants")
 public class ApplicantModel extends BaseModel {
+  public enum Suffix {
+    JR("Jr."),
+    SR("Sr."),
+    I("I"),
+    II("II"),
+    III("III"),
+    IV("IV"),
+    V("V");
+
+    private final String displayName;
```

Comment:
**suggestion (non-blocking):** You could also consider calling this `suffix` or `suffixName`, but good with either.

Reply:
gotcha, changed it to `suffixName`